radis/lbl/base.py

```diff
@@ -1670,8 +1671,8 @@ def _calc_noneq_parameters(
                     "number_of_lines": len(df),
                     "isotope": isotope,
                     "neighbour_lines": self.params.neighbour_lines,
-                    "wavenumber_min": self.input.wavenum_min,
-                    "wavenumber_max": self.input.wavenum_max,
+                    "wavenum_min": self.input.wavenum_min,
+                    "wavenum_max": self.input.wavenum_max,
```

Comment:
In your tests, make sure it also works with wavelength inputs. Normally all inputs are converted to wavenumber so it should work, but better test!